x-pack/plugins/ml/public/jobs/new_job/simple/components/general_job_details/_general_job_details.scss

```diff
@@ -20,6 +20,9 @@
     label {
       font-weight: $euiFontWeightRegular;
     }
+    .dedicated-index-label {
+      margin-left: $euiSizeXS;
```

Comment:
Thanks for using the EUI variables, but can you follow the BEM naming structure for the class name (found at the bottom of this page: https://elastic.github.io/eui/#/guidelines/sass)?

Reply:
@cchaos Thanks for the hint. It's our intention to follow the BEM style for all new components, for this component I don't think it's really worth changing this class name alone in the file but it's also not worth changing all the names as the whole thing is marked for a complete react/eui rewrite in the near future (currently on the roadmap for 7.1)
Out of interest, what would you suggest the name should be? is something like `dedicatedIndexCheckbox__label` correct?

Reply:
Yeah totally understandable. I just figured I'd point it out for new classes added so there's less to re-work later.

I think we tend to look at the naming of the encapsulating component (usually the same as the file name) and start with the prefix so it may be verbose, but something like `mlGeneralJobDetails__dedicatedIndexLabel` is how I'd write it out but what you had could work just fine as well as long as it also gets the `ml` prefix: `mlDedicatedIndexCheckbox__label`.

The more unique of a name also means you don't need to worry about nesting so the selector could even be pulled out of the `.general-job-details` selector.